docs/application/tizen-studio/common-tools/device-manager.md

```diff
@@ -37,36 +37,46 @@ To use the Device Manager with a specific device, you must connect the device ov
 <a name="connecting"></a>
 ## Connecting Devices
 
-The Device Manager shows both local devices connected to the system and any configured remote devices:
+Device Manager shows both local devices connected to the system and any configured remote devices:
 
 -   Only connected local devices (both physical and virtual devices) are listed. Disconnected devices automatically disappear from the list. Therefore, the **Connection** switch is disabled.
 -   Remote devices are listed even if the connection is lost. You can turn the connection on and off with the **Connection** switch.
 
-**Figure: Connected devices in the Device Manager**
+**Figure: Connected devices in Device Manager**
 
-![Connected devices in the Device Manager](./media/device_manager_connected_devices.png)
+![Connected devices in Device Manager](./media/device_manager_connected_devices.png)
 
 Connecting a device to a computer depends on the device type:
 
 -   Physical device
 
-    When you connect a local physical device with a USB cable, it automatically appears in the Device Manager.
+    When you connect a local physical device with a USB cable, it automatically appears in Device Manager.
 
 - Virtual device
 
-    To connect a virtual device, click the **Emulator Manager** button (![Emulator Manager](./media/device_manager_emulator_manager_icon.png)), select the virtual device, and click **Launch**.
+    To connect a virtual device, click **Emulator Manager** ![Emulator Manager](./media/device_manager_emulator_manager_icon.png), select the virtual device, and click **Launch**.
 
-    When the emulator finishes booting, the device appears in the Device Manager.
+    When the emulator finishes booting, the device appears in Device Manager.
 
 - Remote device
 
-    To connect a remote device:
+    To connect a remote device, follow these steps:
 
-    1.  Launch the Remote Device Manager by clicking the **Remote Device Manager** button (![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png)).
-    2. Click **Scan Devices** to display a list of devices you can connect to.
-    3. Confirm the IP and port of the remote device and change the **Connection** switch to **ON**.
+    1. In the Tizen Studio menu, select **Tools > Device Manager**.
+	2. In the **Device Manager** window that appears,  click **Remote Device Manager** ![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png). 
+        
+		![Remote Device Manager](./media/remote_device_manager.png)
```

Comment:
I have 2 suggestions for this image

**Suggestion 1:**

2. In the **Device Manager** window that appears,  click **Remote Device Manager** ![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png). **Remote Device Manager** window appears. 

    _Remote Device Manager image_

3. In the **Remote Device Manager** window, you can scan,..

**Suggestion 2:**

2. In the **Device Manager** window that appears,  click **Remote Device Manager** ![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png). 

3. In the **Remote Device Manager** window, you can scan,..

    _Remote Device Manager image_

Reply:
As we are following Google Style Guide, it is recommended to avoid repeating procedures. Instead, reference those procedures and link to them, so I have used:
 1. In the Tizen Studio menu, select **Tools > Device Manager**.
 2. In the **Device Manager** window that appears,  click **Remote Device Manager** ![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png). 

For more information, see https://developers.google.com/style/procedures
![Google style guide](https://user-images.githubusercontent.com/36875286/84363409-54b4fb80-abec-11ea-9de0-0f30bd63b97d.PNG)

Please let me know your valuable opinion on the same.

Reply:
My point was the location of the "Remote Device Manager" image and the description for that.

If we click "Remote Device Manager" button ![image](https://user-images.githubusercontent.com/9958374/84452011-a06cb100-ac8f-11ea-9258-f091c27c689c.png), the "Remote Device Manager" window appears.
![image](https://user-images.githubusercontent.com/9958374/84452046-b5e1db00-ac8f-11ea-8598-e6f4495f0799.png)


In current status, the "Remote Device Manager window image" is located before the description for appearance of that.

![image](https://user-images.githubusercontent.com/9958374/84452276-5a641d00-ac90-11ea-9d62-8812c1b1b18f.png)

So I hoped to rearrange that. Would you please let me know your opinion?

Reply:
Thank you, this is a good suggestion. 
I will rearrange the image location.
This point was not clear in the previous comment, https://github.com/Samsung/tizen-docs/pull/1142#discussion_r438495556, so the focus went on the text.